app/models/recipe.rb

```diff
@@ -11,4 +16,19 @@ class Recipe < ApplicationRecord
     reject_if: proc { |attribute| attribute['unit_amount'].blank? }, allow_destroy: true
 
   validates_associated :recipe_ingredients
+
+  private
+
+  def check_dimensions
+    temp_file = image.queued_for_write[:original]
+    if temp_file
+      dimensions = Paperclip::Geometry.from_file(temp_file)
+      width = dimensions.width
+      height = dimensions.height
+
+      if width < 250 && height < 250
+        errors[:image] << ' minimum width: 250px, minimum height: 250px'
```

Comment:
Please use the interface that AR provides -> `errors.add(:image, message)` or something like that.